server/src/infra/entities/user.entity.ts

```diff
@@ -75,4 +76,10 @@ export class UserEntity {
 
   @OneToMany(() => AssetEntity, (asset) => asset.owner)
   assets!: AssetEntity[];
+
+  @Column({ type: 'bigint', nullable: true, transformer: new BigIntTransformer() })
```

Comment:
You shouldn't need a BigIntTransformer now, we changed a typeorm parameter in a previous PR so BigInts will become numbers by default now anyway. Something like parseInt8 in the typeorm parameters

Reply:
Oh lol nice. Was actually talking to Jason about this and we thought it was not yet implemented haha
Great to here it is though!